<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QWidget" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="Bottom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1921</width>
     <height>1081</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="AirPage">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>930</y>
      <width>831</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#AirPage{
	border-top-left-radius:10px;
	border-top-right-radius:10px;
	background-color:white;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="SwitchButton" name="AirSwitch" native="true">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>55</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="AirSubUi">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
background:#FFFFFF;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/new/Air/Air/AirP.png</normaloff>:/new/Air/Air/AirP.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QSlider" name="AirSlider">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前空调温度：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AirTemp">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>100</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="MainPage">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>741</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#MainPage{
	background-color:white;
	border-radius:15px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="AirQuality">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>551</width>
       <height>391</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>220</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>500</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>空 气 质 量 指 数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>560</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Air quality index</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AirQualityP">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>510</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LightPage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>930</y>
      <width>831</width>
      <height>151</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#LightPage{
	border-top-left-radius:10px;
	border-top-right-radius:10px;
	background-color:white;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="SwitchButton" name="AllLightSwitch" native="true">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="LightSubUi">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
background:#FFFFFF;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/new/Led/Led/Brightness.png</normaloff>:/new/Led/Led/Brightness.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QSlider" name="AuxiliarySlider">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>65</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>100</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前辅灯亮度：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AllAuxiliaryValue">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>100</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="AirQualityPage">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>891</width>
      <height>861</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>660</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:13px;
background-color:white
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="TvcoP">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>挥发性有机化合物：适宜</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>140</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TVCO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>70%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:13px;
background-color:white
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>湿度：正常
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Humidity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>30%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="HumP">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:13px;
background-color:white
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="PM25P">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>140</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ParticulateMatter2.5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>颗粒物浓度：低</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>20%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:13px;
background-color:white
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="TempP">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>温度：适宜</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>20°</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>340</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:13px;
background-color:white
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="HchoP">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>甲醛：正常</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>140</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>59%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>660</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:13px;
background-color:white
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="Co2P">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>二氧化碳：正常</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Humidity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>20%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="OutSideTemp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>750</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#OutSideTemp{
	background-color:white;
	border-radius:15px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="TempP2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>户外温度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="OutSideTempLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>27°</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="OutSideHum">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>750</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#OutSideHum{
	background-color:white;
	border-radius:15px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="HumP2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="OutSideHumLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>27°</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>户外湿度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>switchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../other/main.qrc"/>
 </resources>
 <connections/>
</ui>
